src/SDKs/dirs.proj

```diff
@@ -60,6 +61,8 @@
 		<ProjectToBuild Include="$(MSBuildThisFileDirectory)Automation\**\*Tests.csproj" ProjectType="Test"/>
     
 		<ProjectToBuild Include="$(MSBuildThisFileDirectory)Batch\Management\**\*Tests.csproj" ProjectType="Test"/>
+		<ProjectToBuild Include="$(MSBuildThisFileDirectory)Batch\DataPlane\Client\Tools\ObjectModelCodeGeneration\ObjectModelCodeGenerator\ObjectModelCodeGenerator.csproj" ProjectType="Test"/>
```

Comment:
@matthchr we have moved away from the declarative model, so these projects will not make it part of the build system.
Right now we have deliberately made it sure that we build limited target framework projects, we are not honoring random target frameworks for projects to have consistency across our SDK (what we talked earlier)
We can still do it but I would like to avoid as much special casing as possible.
Let's talk separately and we can find a solution for this.